app/Http/Controllers/MainController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\View\View;
+
+class MainController extends Controller
+{
+    public static function menu(string $route): array {
```

Comment:
Do not forget to follow up on these automatic generated comments. During the exam, points will be deducted when we find those in your code